assets/js/components/KeyMetrics/MetricTileLoading.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
This component is needed only because you need to replicate the same layout in the custom dimensions wrapper. This is not needed because we need to use the `MetricTileWrapper` component with the `loading={ true }` property. Then this component can be removed.

Reply:
Good point, updated, thanks!